tests/scalers/redis/redis_sentinel_streams_pending/redis_sentinel_streams_pending_test.go

```diff
@@ -0,0 +1,248 @@
+//go:build e2e
```

Comment:
Rename here as well to `tests/scalers/redis/redis_sentinel_streams/redis_sentinel_streams_pending_entries_test.go`